samza-core/src/main/java/org/apache/samza/operators/MessageStreamImpl.java

```diff
@@ -33,11 +28,20 @@
 import org.apache.samza.operators.functions.SinkFunction;
 import org.apache.samza.operators.spec.OperatorSpec;
 import org.apache.samza.operators.spec.OperatorSpecs;
+import org.apache.samza.operators.util.InternalInMemoryStore;
```

Comment:
It's ok, I can wait for Jagadish's PR - his is more up to date. There are a couple of other changes (like onTimer) that we'll need to merge.